egs/wsj/s5/steps/chaina/internal/get_train_schedule.py

```diff
@@ -72,8 +74,11 @@ def get_args():
                         lstm*=0,0.2,0'.  More general should precede less
                         general patterns, as they are applied sequentially.""")
 
+    parser.add_argument("--num-archives", type=int, default=0, required=True,
+                        help="""The number of repeats...TODO
```

Comment:
I renamed this to num-scp-files.

Reply:
Do you mean I should rename the arg to --num-scp-files?

Reply:
yes.  see the example usage in train.sh.
they are physically stored as scp files now, so that's a more accurate name.